src/sass/components/_header.scss

```diff
@@ -224,3 +224,47 @@ header {
     }
   }
 }
+@media (max-width: 767px) {
+  header {
+    .top-bar {
+      .text-left {
+        flex-grow: 2;
+
+        ul {
+          li {
```

Comment:
please reduce the nesting here.

this will result in `header .top-bar .text-left ul li` selector, which is hard to read and maintain. consider adding a class to `ul`/`li` and target them specifically via `.your-new-ul-class li` or ever `.your-new-li-class`

same applies to selector below